### PR TITLE
Add Myntra testimonial

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

File: _data/users.yml [hunk 1]
```diff
@@ -197,6 +197,17 @@
 - name: Myntra
   logo: /assets/images/logos/myntra.png
   logosmall: /assets/images/logos/myntra-small.png
+  testimonial: |
```
Comment (martint) [inline _data/users.yml:200]: Add an `anchor:` so that clicking on their logo takes you to the testimonial

Comment (mosabua) [reply _data/users.yml:201]: correct.. add after name .. see others like linkedin

File: _data/users.yml [hunk 2]
```diff
@@ -197,6 +197,17 @@
...
+    Myntra is among India's leading e-commerce platforms for fashion, beauty and
```
Comment (mosabua) [inline _data/users.yml:202]: beauty, and

File: _data/users.yml [hunk 3]
```diff
@@ -197,6 +197,17 @@
...
+    lifestyle. An integral part of the Flipkart Group, Myntra brings together
+    technology and fashion to create the best experience for its fashion-forward
+    customers, helping them look good. In alignment with our data-first
+    organizational culture, Trino has been an incredible partner in our efforts
+    to democratize data. Our teams run thousands of queries everyday on
```
Comment (mosabua) [inline _data/users.yml:207]: every day

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _data/users.yml (hunk 1 above)
Comment (bitsondatadev) [reply _data/users.yml:201]: added lol realized this after I pushed

**Pull request merged**